plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotationDescriptor.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.erd.ui.notations;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.jkiss.dbeaver.model.impl.AbstractDescriptor;
+
+public class ERDNotationDescriptor extends AbstractDescriptor {
+
+    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_ID = "id"; // $NON-NLS-N$
+    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_NAME = "name"; // $NON-NLS-N$
+    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_DESCRIPTION = "description"; // $NON-NLS-N$
+    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_DEFAULT = "isDefault"; // $NON-NLS-N$
+    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_NOTATION = "notation"; // $NON-NLS-N$
```

Comment:
Why are these public? They are only used in a single place. Mark them as `public` and remove redundant prefixes. We also already have some of these predefined in the `RegistryConstants` class. Reuse if possible.

```suggestion
    public static final String ATTR_ID = "id"; // $NON-NLS-N$
    public static final String ATTR_NAME = "name"; // $NON-NLS-N$
    public static final String ATTR_DESCRIPTION = "description"; // $NON-NLS-N$
    public static final String ATTR_DEFAULT = "isDefault"; // $NON-NLS-N$
    public static final String ATTR_NOTATION = "notation"; // $NON-NLS-N$
```

I'd also like the `notation` attribute to be renamed to `class` as it's the implementation class.